### DISCUSSION TITLE
compile in format .aab for Android

### DISCUSSION THREAD

**Discussion by josquinet** [airsdk/Adobe-Runtime-Support#896] [Q&A]
Do you know something about whether animate can be compiled in .aab format? I think it will be mandatory from August on Android and I am in the middle of a project (an app) that I will surely finish later ... Can't make the .aab format with the Air SDK?

If you can't ..... can you pass APK to .aab somehow?

Thank you!!

**Comment by ajwfrost** [comment]
Hi

Sadly you can't create an AAB file out of APK files, they're compiled quite differently. The Animate team are aware of our forthcoming updates so I am hopeful that they'll provide the necessary UI elements in their next release (around October time if they're keeping the original cadence, but there are a lot of impacts to businesses due to the current Covid situation so I can't say for sure...)

So in the meantime, this is a good point: we'll again need a way for people to build one thing whilst having the options for another.

Probably we'll just add a new setting into the config file so that you can tell it, regardless of what you ask Animate to create, if it sees a request for "apk-*" then it will instead do an "aab".

Note that you'll still need to create APK files for local testing/debugging, it's just for Play Store uploads that you need the bundle. 

thanks
